packages/builder/src/analytics/index.js

```diff
@@ -23,12 +21,9 @@ class AnalyticsHub {
 
   identify(id) {
     posthog.identify(id)
-    sentry.identify(id)
   }
 
-  captureException(err) {
-    sentry.captureException(err)
-  }
+  captureException(_err) {}
```

Comment:
@aptkingston should this not be using posthog?

Reply:
Posthog isn't really an option in the frontend right now, we don't have the ability to do this. Really we need a new way for detecting this stuff, but any errors that go to Sentry are ignored today, so the above isn't actually a change.